lib/michelson-test-check

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+# Usage: ./kmich run ... --search-final --output json | $0 <path-to-definition>
+#
+# Reads kmichelson json output for unit-tests and checks that all branches have succeeded.
+# Exits 127 if there is a stuck term. Otherwise, exits with the number of failing branches
+
+import json
+import os
+import pyk
+import sys
+
+def findSubtermsByLabel(label, term):
+    if not term['node'] in ['KSequence', 'KApply']: return []
+    if term['node'] == 'KApply':
+        if term['label'] == label: return [term]
+        ret = []
+        for subterm in term['args']:
+            ret += findSubtermsByLabel(label, subterm)
+        return ret
+    return []
+
+def parseKastTerm(jsonString):
+    return json.loads(jsonString)['term']
```

Comment:
@ehildenb These may be useful to add to `pyk`

Reply:
`pyk` has `collect{BottomUp,TopDown}` and `traverse{BottomUp,TopDown}`, as well as `match`, which can be used for `findSubtermsByLabel`.

`pyk` has `readKastTerm` which is similar to `parseKastTerm`, but loads it from a file. Maybe it could be factored out into two functions.